db/migrations/0216__move_content_into_blobs.js

```diff
@@ -1,6 +1,6 @@
 module.exports = () => {
-    const sql = require('../../src/services/sql.js');
-    const utils = require('../../src/services/utils.js');
+    const sql = require('../../src/services/sql');
+    const utils = require('../../src/services/utils');
```

Comment:
I think migrations have their worth, especially since I know people that haven't updated their Trilium in a while. If they ever decide to switch from a really old version of Trilium to Trilium Next they should be able to.